LFS-792: The app name, platform name, and platform version should be displayed to the user on every page

OrdiNeu opened the pull request:
[Jira link with picture of the mockup](https://phenotips.atlassian.net/browse/LFS-792)

OrdiNeu committed:
LFS-792: The app name, platform name, and platform version should be
displayed to the user on every page

marta- commented on modules/homepage/src/main/frontend/src/themePage/Sidebar/sidebar.jsx:
```diff
@@ -99,6 +99,11 @@ const Sidebar = ({ ...props }) => {
+  let platformName = document.querySelector('meta[name="title"]').content;
+  let appName = document.querySelector('meta[name="appName"]').content;
+  let version = document.querySelector('meta[name="version"]').content;
+  var versionInfo = <Typography variant="subtitle2" className={classes.versionText}>{platformName} | {appName} v{version}</Typography>
```
The output is correct, but the variable names aren't. Technically 'cards' is the general platform, and "Cards 4 Kids", "Cards 4 CaRe", "LFS Data Core" are apps on top of it.

OrdiNeu replied:
Resolved in `be76129` :heavy_check_mark:

marta- commented on modules/homepage/src/main/frontend/src/themePage/Sidebar/sidebarStyle.jsx:
```diff
@@ -169,6 +169,10 @@ import {
     },
+    versionText: {
+      textAlign: "center",
+      color: whiteColor
```
The text needs to breathe a little. With long version numbers and/or long app names it gets too close to the edges, and it's also eclipsed by the "Administration" entry above it.
```suggestion
      color: whiteColor,
      margin: theme.spacing(4, 2, 0),
```

OrdiNeu replied:
Resolved in `be76129` :heavy_check_mark:

marta- commented on pom.xml:
```diff
@@ -145,7 +145,8 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <lfs.version>${project.version}</lfs.version>
-    <lfs.name>LFS Data Core</lfs.name>
+    <platform.name>LFS Data Core</platform.name>
+    <app.name>CARDS</app.name>
```
Per previous comment, these need to be swapped

OrdiNeu replied:
Resolved in `be76129` :heavy_check_mark:

OrdiNeu committed:
LFS-792: The app name, platform name, and platform version should be
displayed to the user on every page

- Swap platform and app name
- Apply code style suggestions by @marta-

sashaandjic commented:
These two dates are not shown properly on the chart:
![image](https://user-images.githubusercontent.com/23243244/102400774-636fd700-3fb0-11eb-9064-edd5b0accb29.png)
I am suspecting Date exited and Date withdrawn are affected too, but due to another bug are not shown in the chart (they are conditional fields).

OrdiNeu commented:
> These two dates are not shown properly on the chart:

@sashaandjic Those two are addressed in a different PR: #408 and aren't included in this PR.

sashaandjic reviewed (approved):
Tested in cardiac_rehab and in lfs mode for every single date field. Some with default setting, some with modified date setting.
All good!

sashaandjic commented:
> > These two dates are not shown properly on the chart:
> 
> @sashaandjic Those two are addressed in a different PR: #408 and aren't included in this PR.

The comment was misplaced and is now moved to proper PR [408](https://github.com/ccmbioinfo/lfs/pull/408)

sashaandjic reviewed (approved):
Tested in all 3 modes on all pages as admin and non-admin. All good!
